deploy/src/loadEnv.js

```diff
@@ -144,25 +167,25 @@ if (BRIDGE_MODE === 'ERC_TO_ERC') {
     BRIDGEABLE_TOKEN_SYMBOL: envalid.str(),
     BRIDGEABLE_TOKEN_DECIMALS: envalid.num(),
     DEPLOY_REWARDABLE_TOKEN: envalid.bool(),
-    DPOS_STAKING_ADDRESS: addressValidator(),
     BLOCK_REWARD_ADDRESS: addressValidator(),
```

Comment:
in this mode `BLOCK_REWARD_ADDRESS` is also applicable only if `DEPLOY_REWARDABLE_TOKEN` is `true`.

Reply:
Thanks! Fixed https://github.com/poanetwork/poa-bridge-contracts/pull/228/commits/418580b76c74ffe1640a469027b01d2468d1407d